includes/wcs-order-functions.php

```diff
@@ -250,7 +258,7 @@ function wcs_get_new_order_title( $type ) {
 	$type = wcs_validate_new_order_type( $type );
 
 	// translators: placeholders are strftime() strings.
-	$order_date = strftime( _x( '%b %d, %Y @ %I:%M %p', 'Used in subscription post title. "Subscription renewal order - <this>"', 'woocommerce-subscriptions' ) ); // phpcs:ignore WordPress.WP.I18n.UnorderedPlaceholdersText
+	$order_date = ( new DateTime( 'now' ) )->format( _x( 'M d, Y @ h:i A', 'Order date parsed by DateTime::format', 'woocommerce-subscriptions' ) );
```

Comment:
This was throwing a deprecated phpcs error so I patched it in this branch. This whole function is deprecated since `2.2.0` so there's no need to be too worried about how backwards compatible this replacement is. 

WC replaced their `strftime` with the same approach in https://github.com/woocommerce/woocommerce/pull/31333/files